docs/UserGuide.adoc

```diff
@@ -83,7 +84,7 @@ Shorthand: `l`
 === Editing a person : `edit` / `e`
 
 Edits an existing person in the address book. +
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [b/Birthday(DD/MM/YYYY)] [t/TAG]...`
+Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [b/Birthday(DD/MM/YYYY)] w/WEBSITE [t/TAG]...`
```

Comment:
w/WEBSITE is optional, should surround it with square brackets.

Reply:
good catch. thanks